test/unit/test_LIALogicMkTerms.cc

```diff
@@ -231,8 +214,7 @@ TEST_F(LIALogicMkTermsTest, test_ReverseAuxRewrite) {
     PTRef nested = logic.mkEq(logic.getTerm_IntZero(), logic.mkMod(ite, c));
 
     for (PTRef tr : {term, eq, nested}) {
-
-        PTRef termWithAux = DivModRewriter(logic).rewrite(IteHandler(logic).rewrite(tr));
+        PTRef termWithAux = opensmt::rewritings::rewriteDivMod(logic, IteHandler(logic).rewrite(tr));
```

Comment:
Include `IteHandler(logic).rewrite(tr)` in `Rewritings.h` as well?
-> `rewriteIte(logic, tr)`

Reply:
ITE handling is a bit special, I think, at the moment.

Reply:
OK. What about `Substitutor`, `Rewriter`, `ArithmeticEqualityRewriter`?

Reply:
#740